src/_guides/language/coming-from/js-to-dart.md

```diff
@@ -1585,7 +1585,7 @@ Pure JavaScript doesn't support immutability.
 Dart offers multiple ways to make collections like arrays, sets, or
 dictionaries immutable.
 
-* If the list is a compile-time constant and shouldn't
+* If the set is a compile-time constant and shouldn't
```

Comment:
Actually this section might be better served with a more generic word like collection:

```suggestion
* If the collection is a compile-time constant and shouldn't
```

Reply:
Agree, that will be better — thanks for the correction!